backends/core/models.py

```diff
@@ -249,9 +316,48 @@ def destroy(self):
       Param.destroy(*param_ids)
     self.delete()
 
+  def get_status(self, max_retires=10):
+    key = str(self.pipeline_id) + '_' + str(self.id) + '_' + CACHE_KEY_STATUS
```

Comment:
Here the logic is error prone, in case your update method later on change.
So, it's good practice to make use of the `_build_job_prefix` method.

Reply:
done